src/js/components/parts/availabletests.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+class AvailableTests extends React.Component {
+  constructor() {
+    super();
+    this.clickEvent = this.clickEvent.bind(this);
+  }
+  clickEvent() {
+    this.props.addItem(this.props.data);
+  }
+  checkIfItemAlreadyAdded() {
```

Comment:
The name `checkIfItemAlreadyAdded` suggests that it will return a boolean value but actually it returns a button!